importApp/src/main/java/com/commercetools/dataimport/joyrideavailability/AvailabilityJobConfiguration.java

```diff
@@ -0,0 +1,246 @@
+        });
+        final InputStream inputStream = channelsJsonResource.getInputStream();
+        final List<ChannelDraftDsl> list = reader.readValue(inputStream);
+        return new ListItemReader<>(list);
+    }
+
+    @Bean
+    protected ItemWriter<ChannelDraftDsl> channelsDraftWriter(final BlockingSphereClient sphereClient) {
+        return items -> items.forEach(channelDraft -> sphereClient.executeBlocking(ChannelCreateCommand.of(channelDraft)));
+    }
+
+    @Bean
+    protected ItemProcessor<Product, List<InventoryEntryDraft>> inventoryEntryProcessor(final BlockingSphereClient sphereClient) {
+        return item -> {
+            getChannelsWithInventory(sphereClient);
+            return inventoryEntryListByChannel(item);
+        };
+    }
+
+    @Bean
+    public ItemWriter<List<InventoryEntryDraft>> inventoryEntryWriter(final BlockingSphereClient sphereClient) {
+        return entryLists -> entryLists.forEach(items -> items.forEach(inventoryEntry -> sphereClient.executeBlocking(InventoryEntryCreateCommand.of(inventoryEntry))));
+    }
+
+    @Bean
+    protected ItemProcessor<Product, ProductUpdateCommand> setPriceProcessor(final BlockingSphereClient sphereClient) {
+        return item -> {
+            final ProductProjectionSearch searchRequest = ProductProjectionSearch.ofStaged()
+                    .withQueryFilters(m -> m.id().is(item.getId()))
+                    .withPriceSelection(PriceSelection.of(EUR));
+            final PagedSearchResult<ProductProjection> result = sphereClient.executeBlocking(searchRequest);
+            final ProductVariant masterVariant = result.getResults().get(0).getMasterVariant();
+            final Price priceProduct = masterVariant.getPrice();
+            final MonetaryAmount newAmount = priceProduct.getValue().subtract(MoneyImpl.ofCents(60, EUR));
+            getChannelsByCountry(sphereClient);
+            final List<AddPrice> productAndPricesList= new ArrayList<>();
+            for (Channel channel : channelsToAddPrice) {
+                final PriceDraft pricePerChannel = PriceDraft.of(newAmount).withChannel(channel).withCountry(channel.getAddress().getCountry());
+                for ( ProductVariant productVariant : item.getMasterData().getCurrent().getAllVariants()) {
+                    productAndPricesList.add(AddPrice.of(productVariant.getId(), pricePerChannel));
+                }
+            }
+            return ProductUpdateCommand.of(item, productAndPricesList);
+        };
+    }
+
+    @Bean
+    public ItemWriter<ProductUpdateCommand> setPriceWriter(final BlockingSphereClient sphereClient) {
+        return list -> list.forEach(cmd -> sphereClient.executeBlocking(cmd));
+    }
+
+    private List<InventoryEntryDraft> inventoryEntryListByChannel(final Product item) {
+        final List<InventoryEntryDraft> listInventoryEntryDraft = new LinkedList<>();
```

Comment:
the name `inventoryEntryDraftList` would be better, since it is a list and the list at the start of the name sounds like the verb "to list sth.".